bigquery_etl/static.py

```diff
@@ -58,20 +63,20 @@ def publish(project_id):
             if not os.path.exists(schema_file_path):
                 schema_file_path = None
 
-            description_file_path = os.path.join(table_dir, DESCRIPTION_FILENAME)
-            if not os.path.exists(description_file_path):
-                description_file_path = None
+            metadata_file = os.path.join(table_dir, METADATA_FILE)
+            if not os.path.exists(metadata_file):
+                metadata_file = None
```

Comment:
To match the conventions used in the surrounding code I'd suggest using the variable/argument name `metadata_file_path` here and in the `_load_table()` function.